### ISSUE TITLE
Launch options arguments

### ISSUE THREAD

**Issue by MaartenSwBase** [salomvary/soundcleod#164]
Is there perhaps a possibility that a feature launch option can be added that normal browsers have?
That would make it easier for creating scripts that open soundcloud links in soundcleod.
Something like chrome has `-args htt://soundcloud.com/artist-name/song-name/`

**Comment by salomvary** [comment]
Nice idea, assuming anything that does not start with soundcloud.com should get ignored. Contributions are welcome!

**Comment by jugendhacker** [comment]
I also think it's a nice idea +1 from me!

**Comment by jugendhacker** [comment]
I have some time now, I will try to implement it!

**Comment by jugendhacker** [comment]
@MaartenSwBase Now I see there is actually an option to do this!
Do it with the --base-url [url] flag

@salomvary Is this the right way?

**Comment by salomvary** [comment]
No, `--base-url` (https://ssoundcloud.com by default) is what SoundCleod uses internally for deciding whether a given url points to SoundCloud or is an external one. It's primarily meant to be a developer tool (hence the lack of documentation).

**Comment by jugendhacker** [comment]
OK but when I set the baseurl to google.de it opens the google site on startup!

**Comment by MaartenSwBase** [comment]
@salomvary I was working on another solution, what do you think of putting soundcleod in app tray with a small icon. And be able to set soundcleod as default browser. Then all soundcloud links you click will get routed to soundcleod, and the rest wil get routed to the previous default browser.
I was currently working on this, but I just thought what if you do not like this idea at all then my work would be in vain. I don't want to just create a forked version for my own usage, but working on it with guidelines that you would approve of so this could one day actually be implemented.

**Comment by salomvary** [comment]
@MaartenSwBase setting SoundCleod as the default browser sounds like a scary idea to me.

I suggest you rather look into products that allow choosing different default browsers per domain or some other smart rules. Quick Googling revealed this one for example: https://www.choosyosx.com/. Let me know if that worked for you.

**Comment by salomvary** [comment]
This feature has been implemented in SoundCleod 1.4.0. Launch it like this:

```
/Applications/SoundCleod.app/Contents/MacOS/SoundCleod https://soundcloud.com/captioncat/sad-trombone
```

or 
```
\path\to\SoundCleod\SoundCleod.exe https://soundcloud.com/captioncat/sad-trombone
```
